components/ChatApp.tsx

```diff
@@ -1,103 +1,61 @@
 import * as React from "react";
 import Chat from "twilio-chat";
 
+import admins from "../lib/admins";
+
+const channelAlreadyExist = 50307;
```

Comment:
rename to `ERROR_CODE__CHANNEL_ALREADY_EXISTS` add link to in comment `https://www.twilio.com/docs/api/errors/50307`